scripts/tests/backwards-compatibility-test.sh

```diff
@@ -84,9 +102,8 @@ for fed_version in "${versions[@]}"; do
         has_failure=true
       fi
 
-      # cleanup devimint dir
-      tmpdir=$(dirname "$(mktemp -u)")
-      rm -rf "$tmpdir"/devimint-*
+      # cleair=$(dirname "$(mktemp -u)")
```

Comment:
[nit] Comment got a bit mangled

Reply:
Oh. I messed up. We can fix it in post. :D